resources/rest-service/cloudify/migrations/versions/387fcd049efb_5_0_5_to_5_1.py

```diff
@@ -35,62 +33,55 @@
                                   name='visibility_states',
                                   create_type=False)
 
-Base = declarative_base()
-
-
-class Config(Base):
```

Comment:
note that this class is not used here to create the table, because the table is already created elsewhere. It is only used to insert entries with the ORM (which requires a model to do so).

Instead, we declare just a table, with a `config_table = table([columns])`, and we only declare the table for use with `op.bulk_insert`, again - not for creating it.

So we only there declare the columns that we are actually going to use, we don't need to declare all columns, and we don't need to declare constraints etc either.
Think of that table declaration as just telling sqlalchemy about the schema, to use in a `INSERT INTO config (x, y, z) VALUES (...)` <-- it needs to know the "x y z", but that statement doesn't need to know about unused columns or about constraints.